message_definitions/v1.0/common.xml

```diff
@@ -2862,14 +2862,15 @@
     </enum>
     <!-- parachute action enum -->
     <enum name="PARACHUTE_ACTION">
+      <description>Parachute actions. Trigger release and enable/disable auto-release.</description>
       <entry value="0" name="PARACHUTE_DISABLE">
-        <description>Disable parachute release.</description>
+        <description>Disable auto-release of parachute (i.e. release triggered by crash detectors).</description>
       </entry>
       <entry value="1" name="PARACHUTE_ENABLE">
-        <description>Enable parachute release.</description>
+        <description>Enable auto-release of parachute.</description>
       </entry>
       <entry value="2" name="PARACHUTE_RELEASE">
-        <description>Release parachute.</description>
+        <description>Release parachute and kill motors.</description>
```

Comment:
This is what ArduPilot does.  But I don't know why it's here - there may be reasons an implementation doesn't want to do that.... merging anyway.

@hamishwillee

Reply:
@peterbarker I know ArduPilot and PX4 both do this, but  you make a good point. It probably makes sense that if someone wants to actually keep motors running that is explicit - ie a new enum. We could add if someone wants to implement that?